### PR TITLE
Fix style for expiring date quick buttons

### PR DESCRIPTION
**Pull request by belljohan3**

### CODE REVIEW

File: cms/src/components/content/EditContentBasic.vue
Comment (laurensgroeneveld) [inline cms/src/components/content/EditContentBasic.vue:1]: There's a linter warning here now, just be sure to remove this import:

```
'ChevronLeftIcon' is defined but never used
```

Comment (belljohan3) [reply cms/src/components/content/EditContentBasic.vue:1]: This is now done.

File: cms/src/components/content/EditContentBasic.vue [hunk 1]
```diff
@@ -307,10 +308,10 @@ const publishStatus = computed({
                     <LButton
                         type="button"
                         name="6"
-                        variant="custom"
+                        variant="secondary"
                         class="flex-1"
                         size="lg"
```
Comment (laurensgroeneveld) [inline cms/src/components/content/EditContentBasic.vue:313]: Please remove this one, as it's the only one on size `lg` and it causes the other ones to increase in size as well. The base size is fine I think.

```suggestion
```

**Review by laurensgroeneveld — APPROVED** [review]
Good fix and big improvement by using the secondary button I think 👍 We can look into other solutions like a button group in the future.

**Pull request merged**